src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  errorMessage: string;
+};
+
+export const Alert: React.FC<Props> = ({ errorMessage }) => {
+  const [hasAlert, setHasAlert] = useState(true);
+
+  const closeAlert = () => {
+    setHasAlert(false);
+  };
+
+  useEffect(() => {
+    const timerID = setTimeout(closeAlert, 3000);
+
+    return () => {
+      setHasAlert(false);
+      clearTimeout(timerID);
+    };
```

Comment:
```suggestion
    return () => {
      clearTimeout(timerID);
    };
```

Reply:
I see - closeAlert already does that. -Done